packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -20,7 +20,7 @@ import { Typography } from '../Typography';
  * ComboboxInput
  * -----------------------------------------------------------------------------------------------*/
 
-export interface ComboboxProps
+interface ComboboxProps
```

Comment:
`defaultTextValue` in the `Omit` down below shouldn't be optional

Reply:
I've just realised how fucked up this type actually is.... 🤦🏼‍♀️